lib/trento/application/projectors/check_result_projector.ex

```diff
@@ -133,15 +135,25 @@ defmodule Trento.CheckResultProjector do
         _
       ) do
     Enum.each(hosts, fn host_id ->
-      TrentoWeb.Endpoint.broadcast("monitoring:clusters", "checks_results_updated", %{
-        cluster_id: cluster_id,
+      view_data = %{
         host_id: host_id,
-        hosts_executions: [%{cluster_id: cluster_id, host_id: host_id, reachable: true, msg: ""}],
+        cluster_id: cluster_id,
+        reachable: true,
```

Comment:
Nice! Now the magic happens in the view!

By the way, why have you decided to use a variable here, when in the others the data goes direclty in the `render` function